middleware/packet-forward-middleware/packetforward/types/codec.go

```diff
@@ -23,7 +22,7 @@ func init() {
 	// Register all Amino interfaces and concrete types on the authz Amino codec
 	// so that this can later be used to properly serialize MsgGrant and MsgExec
 	// instances.
-	RegisterLegacyAminoCodec(authzcodec.Amino)
+	// RegisterLegacyAminoCodec(authzcodec.Amino) // TODO(bez): Investigate this.
```

Comment:
"Messages no longer need to implement the LegacyMsg interface ... global legacy Amino codec definitions and their registration in init() can safely be removed as well." from [migration docs](https://docs.cosmos.network/main/build/migrations/upgrading#modules-1)

Reply:
Since this is not a hard requirement, I am going to leave this as is. Can remove in the future. 

Feels odd that `RegisterLegacyAminoCodec` still being required from module.AppModuleBasic interface despite its removal.